packages/sdk-js/src/feature-repository.ts

```diff
@@ -428,20 +461,30 @@ async function fetchFeatures(
             error: e ? e.message : null,
           });
         activeFetches.delete(cacheKey);
-        return Promise.resolve({});
+        const existing = !cacheSettings.disableLocalCache
+          ? cache.get(cacheKey)
+          : undefined;
```

Comment:
Should we default to `instance.getPayload()` instead?  Thinking about Next14 SSR, we will disable local cache.  If you call `gb.refreshFeatures()` and the fetch fails, we wouldn't want to overwrite the instance's existing payload with an empty object.

Reply:
would something like this work where it's always plugging in getPayload() by default?
```
!cacheSettings.disableLocalCache
          ? cache.get(cacheKey)
          : undefined
          ?? instance.getPayload();
```